crates/rolldown/tests/esbuild/loader/loader_data_url_unknown_mime/artifacts.snap

```diff
@@ -0,0 +1,24 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/loader/loader_data_url_unknown_mime
+---
+# Assets
+
+## entry_js.mjs
+
+```js
+
+//#region <data:some/thing;what,someData%31%32%33>
+var thing_what_someData_31_32_33__default = "data:some/thing;what,someData%31%32%33";
```

Comment:
See the description. It may be too complex to align with `esbuild` in this PR, imo.

Reply:
Btw, `vite` and `rollup` report errors when handling this custom mime, but in `esbuild` it remains as the import argument. I wonder what the result should be.